woocommerce/google-listings-and-ads
# Streamline onboarding: Clean up the access to the pre-launch checklist in the `mc/settings` API

### Changes proposed in this Pull Request:
This PR removed any backend related to a pre-launch checklist.

Closes #2494.

The PR addresses it's issue. Check screenshots section for after the change screenshots



### Screenshots:
After:
All pre-launch checklist attributes removed from API
![Screenshot 2024-08-22 at 09 44 58](https://github.com/user-attachments/assets/e496379a-6439-4b2e-a4a0-e6c8e43c3549)

We are also able to move to last screen after the removal.
![Screenshot 2024-08-22 at 10 44 18](https://github.com/user-attachments/assets/7558fae1-94ab-4e64-a399-4fb21835d40b)


### Detailed test instructions:


1. Switch to this branch and run build command `npm run build`
2. Delete `gla_merchant_center` saved options use `npm run wp-env run cli wp option delete gla_merchant_center` 
3. On the 3rd screen, verify the Phone number (if not already), add the address(if not already), then press continue; it should take you to the next screen. You should also be able to perform the next steps and create a campaign successfully. 


### Additional details:


### Changelog entry
